.github/workflows/tests.yml

```diff
@@ -296,7 +296,5 @@ jobs:
           woke-args: "*.md **/*.md -c https://github.com/canonical-web-and-design/Inclusive-naming/raw/main/config.yml"
 
       - name: Run link checker
-        # This can fail intermittently due to external resources being unavailable.
```

Comment:
Please can we keep this in so that if an external web site is down/unreachable it doesnt prevent the rest of the tests from running. I added this because we periodically get this job fail because the web site in question is down/unreachable.

Reply:
See the commit message - this is now the last of the test, so all other tests will run. If we add further tests, we can add them with "always()" so they still run.
If we don't fail this test, we will NEVER see if a link breaks, and we can just as well get rid of the test.

Reply:
Does not show it as red/failed in the test results then?

Reply:
Yes, the linkcheck will show as red - but that shouldn't prevent from merging?

Reply:
OK will merge this now and if we see it causing issues again in the future we can review then.